### DISCUSSION TITLE
Can't include access token to Session

### DISCUSSION THREAD

**Discussion by shellingford330** [nextauthjs/next-auth#3473] [Help]
### Question 💬

In V3, the access token was included in the DefaultSession, but in V4, the Adapter API has been simplified. (https://github.com/nextauthjs/next-auth/pull/2361/files)

Thank you.
However, I use the adapter to connect to my own API. That API returns an access token (NOT from provider), but how do I include the access token in the Session?

I don't think it's appropriate to use Callback to query the API and get the access token.
This is because the API should return the access token only at login (create session).

I would like to be able to extend the Session object.

### How to reproduce ☕️

Explained in the description.

### Contributing 🙌🏽

Yes, I am willing to help answer this question in a PR

**Comment by ceres-quangd** [comment]
I had a similar situation. waiting a solution in v4
